### ISSUE TITLE
pactmaker hex's are not scaling at all, also chimera sorcerer is not giving past bloodlines when picked up at lvl 5

### ISSUE THREAD

**Issue by beanbou** [JohN100x1/WOTR_IsekaiMod#142]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Additional context**

**Comment by JohN100x1** [comment]
I'm pre-occupied at the moment. I'll take a look soon.

**Comment by kjk001** [comment]
I will check on the hex as that would be a bug.

The other thing is not a bug but known limitation that is actually stated in the mod description.

Though given that this is the second time this has been reported I think it needs to be added to the actual legacy description as well as a warning.

**Comment by Mythalar** [comment]
Regarding sorcerer/oracle legacies perhaps the lvl 4 selection could be postponed to miss only one as dual legacies (the first) and not 2?
It hurts the oracle quite more since you can choose revelations only after you the corresponding mystery.

**Comment by JohN100x1** [comment]
I'll have a new thread to investigate the limitations a bit more #144. But it probably won't be anytime soon since I'm focusing on localisation for the next update.
